docs/src/developer/reference/config-options.md

```diff
@@ -518,6 +518,18 @@ limitedEventFanout:
     status: disabled
 ```
 
+### Domain Registration
+
+This feature flag is per default disabled and locked. It can be set by `ibis` via the internal API and it is not meant to be configured via team management.
```

Comment:
"set" meaning "changes to enabled / unlocked"?  or does the team admin have to enable it?

Reply:
It can be dis/-enabled and (un-)locked by ibis. Team admin cannot change it.